bugsnag-plugin-android-ndk/src/main/jni/utils/serializer.c

```diff
@@ -242,6 +242,12 @@ void bsg_serialize_context(const bugsnag_event *event, JSON_Object *event_obj) {
   }
 }
 
+void bsg_serialize_grouping_hash(const bugsnag_event *event, JSON_Object *event_obj) {
+  if (strlen(event->grouping_hash) > 0) {
```

Comment:
Most of the other fields are set regardless of whether there is a value in `event` - is there a reason this is guarded?

Reply:
A groupingHash should only be serialized if a user has set the value on the struct. If the value is `NULL` then I believe the current JSON implementation won't serialize it anyway but it felt good to be explicit on this point.